examples/petclinic/service.bal

```diff
@@ -0,0 +1,596 @@
+import petclinic.db;
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/persist;
+import ballerina/sql;
+
+configurable int port = 9966;
+
+@http:ServiceConfig {
+    cors: {
+        allowOrigins: ["*"],
+        allowCredentials: false,
+        allowHeaders: ["*"],
+        allowMethods: ["GET", "POST", "PUT", "DELETE", "OPTIONS"]
+    }
+}
+service /petclinic/api on new http:Listener(port) {
+
+    private final db:Client dbClient;
+
+    function init() returns error? {
+        self.dbClient = check new ();
+        log:printInfo("Pet Clinic API started successfully on port " + port.toString());
+    }
+
+    isolated resource function get owners(string? lastName = ()) returns db:Owner[]|InternalServerErrorProblem {
+        do {
+            sql:ParameterizedQuery whereClause =
+                lastName is string
+                ? `last_name = ${lastName}`
+                : ``;
+            stream<db:Owner, persist:Error?> ownersStream = self.dbClient->/owners.get(whereClause = whereClause);
+
+            db:Owner[] owners = check from db:Owner owner in ownersStream
+                select owner;
+
+            log:printDebug(string `Retrieved ${owners.length()} owner(s)` + (lastName is string ? string ` with last name: ${lastName}` : ""));
```

Comment:
We can use the raw template without mentioning it as string template:
```suggestion
            log:printDebug(`Retrieved ${owners.length()} owner(s)` + (lastName is string ? string ` with last name: ${lastName}` : ""));
```

Check other places as well

Reply:
I see, the concatenation is not supported in the log raw templates. So lets have it as `string` template for now